bodhi/server/consumers/automatic_updates.py

```diff
@@ -157,7 +141,7 @@ def __call__(self, message: fedora_messaging.api.Message) -> None:
                 stable_karma=3,
                 unstable_karma=-3,
                 user=user,
-                status=UpdateStatus.testing,
+                status=UpdateStatus.pending,
```

Comment:
does the request have to be set to testing here too? or does that not matter?

Reply:
The request bit does only really matter when the release is composed by Bodhi (which is not the case for rawhide). 
Bodhi's composer which runs every night query all the updates that are in pending status with the request set to testing, the list of update that is returned will then be composed (build added to the update-testing repo). The same is true for updates in testing status with the request set to stable.

So in the case of rawhide we don't have to worry about the request.